src/alitra/models/bounds.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass
+
+from alitra.frame_dataclasses import Point
+
+
+# Point 1 and 2 should be on the diagonal corners of the bounding cube
+@dataclass
+class Bounds:
+    point1: Point
+    point2: Point
```

Comment:
Is point1 and point2 well described names?
Alternatively lower_bound and upper_bound or similar?